course_1_1/Program.cs

```diff
@@ -10,19 +10,331 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Please enter a radius");
-            string rstr = Console.ReadLine();
-            int.TryParse(rstr, out int r);
-            Console.WriteLine($"The square of the round {Program.PerformCalculation(r)}");
-            
+            //
+            //HoursInDay();
+            //
+            //
+            //ABetweenC();
+            //
+            //
+            //OddThreeDigit();
+            //
+            //
+            //XYZ();
+            //
+            //
+            //Calculator();
+            //
+            //
+            //HittingGap();
+            //
+            //
+            //Translate();
+            //
+            //
+            //Payment();
+            //
+            //
+            //CalculatorWithRepeat();
+            //
+            //
+            Factorial();
+            //
         }
 
-        public static decimal PerformCalculation(int r)
+        public static void HoursInDay()
         {
-            const decimal Pi = 3.14m;
-            decimal square = Pi * r * r;
-            return square;
-            
+            Console.WriteLine("Please enter a seconds");
+            string strseconds = Console.ReadLine();
+            int.TryParse(strseconds, out int seconds);
```

Comment:
I'd suggest to use full names in methods instead of aliases.

Reply:
Also TryParse returns a value that indicates if the parsing was succeeded.
So to avoid incorrect logic we need check for that too.